js/core/seo.js

```diff
@@ -0,0 +1,55 @@
+
+// Module core/seo
+// Manages SEO information for documents
+// e.g. set the canonical URL for the document if configured
+
+define(
+    ["core/pubsubhub"],
+    function (pubsubhub) {
+        return {
+            run:    function (conf, doc, cb) {
```

Comment:
Nit: one space here. I've been trying to adhere to standard js for style. Same with other white space: max two spaces of indentation.